modules/aws/sn-volume-access/files/sn_volume_s3_bucket.json.tpl

```diff
@@ -0,0 +1,28 @@
+{
+    "Version": "2012-10-17",
+    "Statement": [
+        {
+            "Effect": "Allow",
+            "Action": [
+                "s3:ListBucket",
+                "s3:PutObject",
+                "s3:GetObject",
+                "s3:DeleteObject"
```

Comment:
`PutObject`, `GetObject`, `DeleteObject` should work with `"arn:aws:s3:::${bucket}/${path}/*"`

Reply:
Thanks, fixed